src/sidebar/components/Settings/Integrations/ApiConfigurationModal/index.tsx

```
@@ -0,0 +1,204 @@
+    const { isOpen, onOpenChange } = useDisclosure()
+    const { t } = useTranslation()
+
+    const [input, setInput] = React.useState('')
+    const [showApiKey, setShowApiKey] = React.useState(false)
+
+    useEffect(() => {
+      setInput(integration?.apiKey ?? '')
+    }, [integration])
+
+    const onSave = async () => {
+      if (integration && integration.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)